tests/tests.fish

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env fish
```

Comment:
So, the tests are explicitly written assuming that you have bash, and can trigger the other shells from there. As in, the tests are meant to all run from one file, that tests all envs. You created a separate file for fish tests, which is not the intent. Was there a reason this needed to happen?